opendap/src/main/java/opendap/servlet/AbstractServlet.java

```diff
@@ -413,19 +413,22 @@ public void IOExceptionHandler(IOException e,  ReqState rs) {
    * @param e        The exception that caused the problem.
    * @param rs The <code>ReqState</code> for the client.
    */
-  public void anyExceptionHandler(Throwable e, ReqState rs) {
-    HttpServletResponse response = rs.getResponse();
+  public void anyExceptionHandler(Throwable e, ReqState rs)
+  {
+    log.error("DODServlet ERROR (anyExceptionHandler): " + e);
+    e.printStackTrace();
+    printThrowable(e);
```

Comment:
I dislike adding new code that prints to STDOUT or STDERR. Why not just use [this method](https://www.slf4j.org/api/org/slf4j/Logger.html#error(java.lang.String,%20java.lang.Throwable)), which will print the entire exception message and stack trace to the log? Better still would be if you removed *all* printing to STDERR from the whole class.